cgroup1/utils.go

```diff
@@ -137,7 +137,7 @@ func readUint(path string) (uint64, error) {
 	}
 	defer f.Close()
 
-	b := make([]byte, 128) // Chose 128 as some files have leading/trailing whitespaces and alignment
+	b := make([]byte, 32) // Chose 32 as some files have leading/trailing whitespaces and alignment
```

Comment:
I'd just plop the comment on my old pr here. It seems for every cgroup file there's just a single line feed at the end and not trailing whitespace, and we expect to be able to be able to read a uint64 in 20 bytes etc.